packages/rspack-dev-server/package.json

```diff
@@ -5,6 +5,12 @@
   "description": "Development server for rspack",
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
+  "exports": {
```

Comment:
`"exports": "./dist/index.js"` should just work 🤔 ?

Reply:
we may add more entryPoint, so I use the more general form

Reply:
I want a publint, does oxclint support now?

Reply:
> we may add more entryPoint later, so I use the more general form

You can always change it later?

> I want a publint, does oxclint support now?

no.

Reply:
actually, it also needs add type field or typescript will not work, so `"exports": "./dist/index.js"` seems not enough here